be/src/olap/storage_engine.cpp

```diff
@@ -873,7 +873,10 @@ void StorageEngine::start_delete_unused_rowset() {
     {
         std::lock_guard<std::mutex> lock(_gc_mutex);
         for (auto it = _unused_rowsets.begin(); it != _unused_rowsets.end();) {
-            if (it->second.use_count() == 1 && it->second->need_delete_file()) {
+            uint64_t now = UnixSeconds();
```

Comment:
```suggestion
            uint64_t now = ::time(nullptr);
```
May be more efficient (40ns vs 4ns)